src/main/groovy/life/qbic/utils/NanoporeParser.groovy

```diff
@@ -171,6 +177,24 @@ class NanoporeParser {
             return convertedDirectory
         }
 
+        private static Map convertToRelativePaths(Map content, Path root) {
+            content["path"] = toRelativePath(content["path"] as String, root)
+            if (content["children"]) {
+                // Children always contains a map, so convert recursively
+                content["children"] = (content["children"] as List).collect { convertToRelativePaths(it as Map, root) }
+            }
+            return content
+
+        }
+
+        private static String toRelativePath(String path, Path root) {
+            if (root.toString() == path) {
+                return "./"
+            } else {
+                return path.replace(root.toString(), "")
```

Comment:
Do not create relative paths with a path separator. They look like absolute paths.
```suggestion
                String relativePath = path.replace(root.toString(), "")
                relativePath = relativePath.startsWith(File.Separator) ? relativePath.substring(1) : relativePath
                return relativePath
```

Reply:
What is the status of this? Any reasons not to adopt this?